client/apollo/react/src/Form/checkbox/checkbox/CheckboxCommon.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Svg } from "../../../Svg/Svg";
+
+type CheckboxIcon = {
+  checkBoxIcon: string;
+};
+
+export type CheckboxProps = {
+  inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
+  inputId?: string;
+  idError?: string;
+  isError?: boolean;
+  className?: string;
+  onChange?: React.ChangeEventHandler;
+} & Omit<React.InputHTMLAttributes<HTMLInputElement>, "label" | "disabled">;
+
+export type CheckboxCommonProps = CheckboxProps & CheckboxIcon;
+const CheckboxCommon = ({
+  inputProps,
+  inputId,
+  idError,
+  checkBoxIcon,
+  isError,
+  onChange,
+}: CheckboxCommonProps) => (
```

Comment:
pas besoin de prop `inputProps` si tu fais de la destructuration

```suggestion
const CheckboxCommon = ({
  inputId,
  idError,
  checkBoxIcon,
  isError,
  onChange,
  ...inputProps
}: CheckboxCommonProps) => (
```